src/App.tsx

```diff
@@ -1,22 +1,34 @@
 import { lazy, Suspense, useEffect } from "react";
 
 import { BrowserRouter as Router } from "react-router-dom";
+import { useMount } from "react-use";
 
 import LoadingSpinner from "@app/components/atoms/LoadingSpinner/LoadingSpinner";
 
+import { getMe, getTokens } from "./features/auth/auth";
 import { useLocalization } from "./features/localization/localization";
 import {
   PermissionEnum,
   setPermissions,
 } from "./features/permissions/permissions";
-import { useAppDispatch } from "./redux/store";
+import { useAppDispatch, useAppSelector } from "./redux/store";
 
 // Routes are lazy loaded so they will access to correct permissions
 const Routes = lazy(() => import("./routes/Routes"));
 
 const App = () => {
   const { loadingTranslation } = useLocalization({ shouldCall: true });
+  const { accessToken } = getTokens();
   const dispatch = useAppDispatch();
+  const { isAuthenticated } = useAppSelector(state => ({
+    isAuthenticated: state.auth.isAuthenticated,
+  }));
```

Comment:
It can be simpler
```suggestion
  const { isAuthenticated } = useAppSelector(state => state.auth);
```

Reply:
Well then you are changing what will trigger a re-render so yes it is shorter, but is it better 😉 

```tsx
// Only re-renders when state.auth.isAuthenticated changes
const { isAuthenticated } = useAppSelector(state => ({
    isAuthenticated: state.auth.isAuthenticated,
}));

// Re-renders when `state.auth` has changed
const { isAuthenticated } = useAppSelector(state => state.auth);
```